tslint.json

```diff
@@ -21,7 +21,7 @@
         ],
         "quotemark": [
             true,
-            "double"
+            "single"
```

Comment:
Yes, in js community the trend is to go for single quote, the reasoning behind this is that you already use double quotes in the HTML, 

https://github.com/airbnb/javascript/issues/110

Checking some source code on some popular projects:

https://github.com/facebook/react/blob/master/src/renderers/dom/fiber/ReactDOMFiber.js

https://github.com/reactjs/redux/blob/master/src/createStore.js

I think the main point is to be consistent, one way or the other, but it seems that the community is going for single quotes.